docs/iota-did-method-spec.md

```diff
@@ -0,0 +1,128 @@
+# IOTA DID Method Specification
+
+Version 0.1-draft by Jelle Millenaar, IOTA Foundation
+
+## Abstract
+
+The IOTA DID Method Specification describes a method of implementing the [Decentralized Identifiers](https://www.w3.org/TR/did-core/) (DID) standard on the IOTA Tangle, a Distributed Ledger Technology (DLT). It conforms to the [DID specifications v1.0 Working Draft 20200731](https://www.w3.org/TR/2020/WD-did-core-20200731/) and described how to publish DID Document Create, Read, Update and Delete (CRUD) operations to the IOTA Tangle. 
+
+## Introduction
+
+### The IOTA Tangle
+
+This specification defines a method of implementing DID on top of the IOTA Tangle, which is a Distributed Ledger Technology (DLT) using a Tangle data structure. In contrast to a Blockchain, the Tangle does not store messages in blocks and chain them together, but rather creates a data structure where a message references two previous messages, creating a parallel structure. 
+
+For this method, the most important features of IOTA are: 
+
+* The lack of fees, requiring no cryptocurrency tokens to be owned in order to submit a message to the DLT.
+* Pure data messages are possible to be stored immutably. 
+* Few nodes store the entire Tangle, requiring additional logic to prove the immutability of data. 
+
+## DID Method Name
+
+The namestring to identify this DID method is: `iota`.
+
+A DID that uses this method MUST begin with the following prefix: `did:iota`. Following the generic DID specification, this string MUST be completely in lowercase.
+
+## DID Format
+
+The DIDs that follow this method have the following format:
+```
+iota-did = "did:iota:" iota-specific-idstring
+iota-specific-idstring = [ iota-network ":" ] [ network-shard ":" ] iota-tag
```

Comment:
```suggestion
iota-specific-idstring = [ iota-network ":" [ network-shard ":" ] ] iota-tag
```

This way `iota-network` is **required** if using `network-shard`, otherwise using `network-shard` alone makes it unclear if it should be a shard or network.

Reply:
Excellent suggestion, will edit.